### PR TITLE
Updated dockerfile for CS 225 Docker Image

### PR DESCRIPTION
**Pull request by Binary-Eater**
Added valgrind to the docker image.

### CODE REVIEW

File: environments/centos7-cs225/Dockerfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ RUN    yum -y install epel-release \
 	&& yum -y install gcc \
 	&& yum -y install gcc-c++ \
 	&& yum -y install centos-release-scl-rh \
-	&& yum -y install devtoolset-3-gcc devtoolset-3-gcc-c++
+	&& yum -y install devtoolset-3-gcc devtoolset-3-gcc-c++ \
+  && yum -y install devtoolset-3-valgrind
```
Comment (JParisFerrer) [inline environments/centos7-cs225/Dockerfile]: Can we fix this indentation :x

Comment (Binary-Eater) [reply environments/centos7-cs225/Dockerfile]: Oh my vim configs..... Give me a sec

**Comment by Binary-Eater** [comment]
Just fixed the indentation. Probably should wait for Travis still.

**Comment by nwalters512** [comment]
Ignoring travis failure for missing Changelog because this isn't a change to PL itself

Note to self: move Dockerfile into our own PL repo 🙂

**Pull request merged**